suite/performance/flux.go

```diff
@@ -192,20 +199,19 @@ func (f *FluxTest) deployContract(contractChan chan<- contracts.FluxAggregator)
 
 func (f *FluxTest) createChainlinkJobs() error {
 	jobsChan := make(chan FluxJobMap, len(f.chainlinkClients)*len(f.contractInstances))
+	g := errgroup.Group{}
 
 	for _, contract := range f.contractInstances {
 		contract := contract
-		g := errgroup.Group{}
-
 		for _, node := range f.chainlinkClients {
 			node := node
 			g.Go(func() error {
 				return f.createChainlinkJob(contract, node, jobsChan)
 			})
 		}
-		if err := g.Wait(); err != nil {
-			return err
-		}
+	}
```

Comment:
Change this back. I did this specifically as too many requests at once will cause issues with file handles:
```
E0823 11:02:30.144049   25198 portforward.go:306] error accepting connection on port 45607: accept tcp4 127.0.0.1:45607: accept4: too many open files
E0823 11:02:30.144057   25198 portforward.go:306] error accepting connection on port 37949: accept tcp4 127.0.0.1:37949: accept4: too many open files
11:02AM INF Creating Job Node URL=http://127.0.0.1:37949
E0823 11:02:30.144097   25198 portforward.go:306] error accepting connection on port 37631: accept tcp4 127.0.0.1:37631: accept4: too many open files
11:02AM INF Creating Job Node URL=http://127.0.0.1:43789
```

Reply:
Ok, nice that you have linux to check that. Maybe adding more descriptors in CI can work later.